index.js

```diff
@@ -101,10 +101,24 @@ const displayResult = () => {
   }
   // CASE two Hexes
   if (firstColor.length === 7 && secondColor.length === 7) {
-    ratioResult.innerHTML = colorFormatRatio(firstColor, secondColor, hexToRGB);
+    if(hexRegex.test(firstColor) && hexRegex.test(secondColor)){
+       ratioResult.innerHTML = colorFormatRatio(firstColor, secondColor, hexToRGB);
+    };
   }
+  if(firstColor.length >= 7 && secondColor.length >= 7) {
+    if(!hexRegex.test(firstColor) || !hexRegex.test(secondColor)){
+       ratioResult.innerHTML = `
+        <div class = "error-message">
+          <h3>Invalid Input</h3>  
+          <p>The following format is supported. Hexadecimal (e.g #000000, #f1f1f1)</p>
+        </div>`;
+    };
+  }else if(firstColor.length < 7 || secondColor.length < 7) {
```

Comment:
As was said in discord, the check
```
else if(firstColor.length < 7 || secondColor.length < 7)
```
may be not needed

if we modify the return statement in calculatRatio function like this:
```
return ((lighterLum + 0.05) / (darkerLum + 0.05)).toFixed(2) === NaN
                    ? "" : ((lighterLum + 0.05) / (darkerLum + 0.05)).toFixed(2);
```
This is suggestion not change request. Be aware, I just modified it quick and dirty. Additional variable may be appropriate here for readability.
Anybody who will work on #76 should keep this in mind I guess.